frontend/ui/widget/dictquicklookup.lua

```diff
@@ -156,6 +158,15 @@ function DictQuickLookup:isDocless()
     return self.ui == nil or self.ui.highlight == nil
 end
 
+function DictQuickLookup:getHtmlDictionaryCss()
+    -- Using Noto Sans becaue Nimbus doesn't contain the IPA symbols.
+    local css = "@page { margin: 0; font-family: 'Noto Sans'; } body { margin: 0; } "
```

Comment:
Might be clearer as a multiline string with `[[]]`?

Reply:
Yeah, I wasn't sure about this. It's indeed clearer with the multiline string.